manifests/prerequisites.pp

```diff
@@ -31,7 +31,7 @@
             }
         }
         'Debian' : {
-            if ($one::params::one_repo_enable == 'true') {
+            if ($one::params::one_repo_enable == true) {
```

Comment:
here is the error with the tests... you turned 'true' to true (boolean) ... and it seems puppet can't handle this.
Look at the discussion in #130 ... there are some problems with booleans and hiera values.